CHANGELOG.mkd

```diff
@@ -1,12 +1,20 @@
 # OpenXC iOS App Demo Changelog
 
-## v5.0.0
+## v6.0.0
+
+* Improvement: Support for Protobuf 3
+* Feature: Added disable trace loop and restart trace features
+* Feature: Added base64 encoding to source ID for cloud export
+* UI: Minor UI layout changes
+* Fix: Updated code signing data
+* Fix: Replaced numerous nil values with default values
+* Fix: Added AppTransportSecurity setting
```

Comment:
Remove below two lines:
Fix: Added AppTransportSecurity setting.(This is part of Upload trace to URL)
Fix: Updated code signing data(our Provisioning profile expired it not a fix .Every one has their own developer certificate and profile .)

Reply:
I updated the change log